src/records/Project.js

```diff
@@ -20,6 +20,7 @@ export default class Project extends Record({
   updatedAt: null,
   instructions: '',
   externalLocations: new ExternalLocations(),
+  archived: false,
```

Comment:
Prefer `isArchived` for boolean properties

Reply:
Yup!